src/js/components/MaskedInput/MaskedInput.js

```diff
@@ -368,7 +368,7 @@ if (process.env.NODE_ENV !== 'production') {
   MaskedInputDoc = require('./doc').doc(MaskedInput); // eslint-disable-line global-require
 }
 const MaskedInputWrapper = compose(
-  withFocus,
+  withFocus({ focusWithMouse: true }),
```

Comment:
I'm not loving the focus on MaskedInput due to the bottom focus-border, need a second opinion here @ericsoderberghp.
![screen shot 2019-01-23 at 9 00 48 pm](https://user-images.githubusercontent.com/6320236/51654062-604e4680-1f53-11e9-873a-6068d0037fca.png)

Reply:
Would it be better if the suggestions wrapper was also highlighted with a green border?

Reply:
@petercpwong I don't think so. 
@ericsoderberghp what do you think?

Reply:
I would ask @L0ZZI.